README.md

```diff
@@ -0,0 +1,22 @@
+# WTExtractor
+Tools to work with WildTanget game assets
+## Pywtools
+Requires Python 3, pip, and cabextract.
+
+    sudo apt update
+    sudo apt install python3 python3-pip cabextract -y
+### Decompile wtxt to txt
+    git clone https://github.com/diamondman/WTExtractor.git
+    cd WTextractor/pywtools/
+    pip install -r requirements.txt
+    ./wtextract.py /path/to/wtxtfile.wtxt /path/to/newtxtfile.txt
+
+Not specifying a txt file will output its text in the terminal.
+    
+## Compile txt to wtxt
+This requires WildCompress, which is only available on Windows.
+* Download the WildTanget SDK under releases, and navigate to SDK/tools/ and run WildCompress Setup.exe to install WildCompress
```

Comment:
I am checking if the guy who gave me the SDK is ok with me putting it up for general download. If so, we can make the instructions clearer by providing the link!